spec/models/application_record_spec.rb

```diff
@@ -104,6 +104,21 @@
     end
   end
 
+  describe 'interview_count' do
+    before :each do
+      record_1 = create :application_record
+      record_2 = create :application_record
+      create :interview, application_record: record_1
+      @collection = ApplicationRecord.where(id: record_1.id)
```

Comment:
@Anbranin's code is correct - this is testing an ActiveRecord scope, so it needs to be called on an ActiveRecord relation.

One application record has an interview, one does not - the scope should find that only one does and return the number 1.